FinancialTracker/Helpers/Currency.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  Currency.swift
+//  FinancialTracker
+//
+//  Created by Boyan Pavlov on 22.01.22.
+//
+
+import Foundation
+
+struct Links {
+    static let symbolCurrencyApi = "https://raw.githubusercontent.com/mansourcodes/country-databases/main/currency-details.json"
+    static let ratesCurrencyApi = "https://open.er-api.com/v6/latest"
+}
+
+class Currencies {
+    var allCurrencies: [Currency] = []
+    
+    init() {
+        let sem = DispatchSemaphore.init(value: 0)
```

Comment:
Why are you using a semaphore?

Reply:
I am using a semaphore because i need to fetch the data first and afterwards show the balance view controller. Because there is like a gap of 0.1< seconds in which the controller has already loaded.